vortex-array/src/view.rs

```diff
@@ -1,13 +1,19 @@
 use std::fmt::{Debug, Formatter};
 
+use enum_iterator::all;
 use itertools::Itertools;
 use vortex_buffer::Buffer;
-use vortex_dtype::DType;
+use vortex_dtype::flatbuffers::PType;
+use vortex_dtype::half::f16;
+use vortex_dtype::{DType, Nullability};
 use vortex_error::{vortex_bail, vortex_err, VortexError, VortexResult};
+use vortex_scalar::flatbuffers::Primitive;
```

Comment:
Try to use `fb::` prefix for these, otherwise it gets very confusing very quickly! Same for flatbuffers::PType above